src/Processing.js

```diff
@@ -3235,7 +3235,7 @@
         b = p.lerp(hsb1[2], hsb2[2], amt);
         rgb = p.color.toRGB(h, s, b);
         // ... and for Alpha-range
-        a = p.lerp(a1, a2, amt) * colorModeA;
+        a = p.round(p.lerp(a1, a2, amt) * colorModeA);
```

Comment:
Just a lil' note: arithimetic operators got higher precedence over the other binary 1s.  
So parens can be removed: `x+.5 | 0`.  :P